### PR TITLE
feat(epmedutour-10477): Implement "Как работает приложение" section

### PR DESCRIPTION
**Pull request by ashumski-adnoc**
### What does this PR do:
Implementation of "Как работает приложение" section

#### Visual change - screenshot after:
Screenshot 2022-08-09 at 10 17 09
Screenshot 2022-08-09 at 10 18 20

#### Ticket Links:

https://jira.epam.com/jira/browse/EPMEDUTOUR-10383

### CODE REVIEW

**Review by mariakravchenko — APPROVED** [review]

**Review by alexeykomov — COMMENTED** [review]
Please use `<picture>` element and serve screenshots in `webp`, `jpeg` formats.

Current format for screens ([1](https://github.com/radzima-green-travel/radzima.app/pull/39/files#diff-5502a53280255527761c29a4dbdd012fa6397129e8e4fc461e6f9f272bae1c8e),[2](https://github.com/radzima-green-travel/radzima.app/pull/39/files#diff-9788f72ff52551d393bc4d664fb1ae82c0097bfb242ee55616b7b647950566dc),[3](https://github.com/radzima-green-travel/radzima.app/pull/39/files#diff-ecb4c0f3a3e93ef482af730d86c2ef5818d2ed7b8598cd6ecd3ce2c8873dd112)) is png, which is lossless. So we have 487, 184, 199 KB size pictures.

Example of `<picture>`:
```
<picture>
<source type="image/webp" srcset="img/avatar1x.webp 1x, img/avatar1.5x.webp 1.5x, img/avatar2x.webp 2x, img/avatar3x.webp 3x">
<source type="image/jp2" srcset="img/avatar1x.jp2 1x, img/avatar1.5x.jp2 1.5x, img/avatar2x.jp2 2x, img/avatar3x.jp2 3x">
<source type="image/jpeg" srcset="img/avatar1x.jpg 1x, img/avatar1.5x.jpg 1.5x, img/avatar2x.jpg 2x, img/avatar3x.jpg 3x"><img src="img/avatar1x.jpg" alt="Avatar" class="avatar">
</picture>
```

File: src/assets/js/index.js [hunk 1]
```diff
@@ -38,4 +38,58 @@ document.addEventListener("DOMContentLoaded", () => {
...
+      });
+    } else {
+      [].forEach.call(document.getElementsByClassName('c-app__grid-cell'), function(elem) {
+        elem.addEventListener('click', function() {
+          const path = `../../images/${this.id.replace('description-','')}.png`;
+          const img = document.getElementById('screenshot-app');
+
+          img.src = path;
+
+          Array.from(screenshotList).forEach((elem) => (
```
Comment (alexeykomov) [inline src/assets/js/index.js:87]: `Array.from` is [not supported](Array.from) in `IE11`. Maybe we can use `Array.prototype.slice`?

File: src/assets/js/index.js [hunk 2]
```diff
@@ -38,4 +38,58 @@ document.addEventListener("DOMContentLoaded", () => {
...
+    const appScreenshotCarousel = document.querySelector('.c-app__imgs');
+    const appScreenshot = document.querySelector('.c-app__desktop-img');
+    const screenshotList = document.getElementsByClassName('c-app__grid-cell');
+    const mql = window.matchMedia('(max-width: 400px)');
+
+    if (mql.matches) {
+      appScreenshot.classList.add('u-hidden');
+      appScreenshotCarousel.classList.remove('u-hidden');
+
+      Array.from(screenshotList).forEach((elem, i) => {
```
Comment (alexeykomov) [inline src/assets/js/index.js:51]: Same about `Array.from`.

**Review by alexeykomov — COMMENTED** [review]
It looks like this element is interactive, but it's a `<div/>`. Please use `<button type="button" />`.

Screenshot 2022-08-12 at 14 46 03

File: src/_pages/ru/index.ejs [hunk 3]
```diff
@@ -49,6 +49,40 @@ title: Главная
...
+      <div class="c-app__desktop-img">
+        <img id="screenshot-app"src="/images/screenshot-app-1.png" alt="screenshot-app">
+      </div>
+      <div class="swiper c-app__imgs u-hidden">
+        <div class="swiper-wrapper">
+          <div class="swiper-slide c-app__img">
+            <img id="screenshot-app-1" src="/images/screenshot-app-1.png" alt="screenshot-app-1">
+          </div>
+          <div class="swiper-slide c-app__img">
+            <img id="screenshot-app-2" src="/images/screenshot-app-2.png" alt="screenshot-app-2">
```
Comment (alexeykomov) [inline src/_pages/ru/index.ejs:63]: In `alt` we should put some description of [what is actually shown on image](https://weblind.ru/inner.html#image-content).

**Pull request merged**